src/game/GameOptions.as

```diff
@@ -38,24 +38,26 @@ package game
         public var displayScore:Boolean = true;
         public var displayCombo:Boolean = true;
         public var displayComboTotal:Boolean = true;
+        public var displayAccuracyBar:Boolean = true;
         public var displayPA:Boolean = true;
         public var displayAmazing:Boolean = true;
         public var displayPerfect:Boolean = true;
         public var displayScreencut:Boolean = false;
         public var displaySongProgress:Boolean = true;
+        public var displaySongProgressText:Boolean = false;
 
-        public var displayMP:Boolean = true;
+        public var displayMPUI:Boolean = true;
         public var displayMPJudge:Boolean = true;
         public var displayMPPA:Boolean = true;
         public var displayMPCombo:Boolean = true;
 
         public var judgeColours:Array = [0x78ef29, 0x12e006, 0x01aa0f, 0xf99800, 0xfe0000, 0x804100];
         public var comboColours:Array = [0x0099CC, 0x00AD00, 0xFCC200, 0xC7FB30, 0x6C6C6C, 0xF99800, 0xB06100, 0x990000, 0xDC00C2]; // Normal, FC, AAA, SDG, BlackFlag, AvFlag, BooFlag, MissFlag, RawGood
         public var enableComboColors:Vector.<Boolean> = new <Boolean>[true, true, true, false, false, false, false, false, false];
-        public var gameColours:Array = [0x1495BD, 0x033242, 0x0C6A88, 0x074B62];
+        public var gameColours:Array = [0x1495BD, 0x033242, 0x0C6A88, 0x074B62, 0x000000];
```

Comment:
CANADA

Reply:
search all "colour"

Reply:
Spelling is currently being annexed, is taking it's time though.